manifests/install.yaml

```diff
@@ -1,23 +1,12 @@
 # This is an auto-generated file. DO NOT EDIT
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   labels:
     app.kubernetes.io/name: applications.argoproj.io
     app.kubernetes.io/part-of: argocd
   name: applications.argoproj.io
 spec:
-  additionalPrinterColumns:
```

Comment:
Curious, why are we removing the printer columns?

Reply:
They're still there! Just moved inside the `versions` array, see: https://github.com/argoproj/argo-cd/pull/5516/files/527dfbef15d9dbe99140b5baeeefeb8f6ed24b01?diff=split&w=1#diff-085adbc8bea5279ae0da31de54e660d34708509979e708a566d8e5f5339822d8R21

Reply:
Great, thanks!